thentos-core/exec/ImportAulaUsers.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Main where
```

Comment:
I can put it in aula. I mostly put it here because it was more convenient to develop that way.